packages/eslint-plugin/src/types.ts

```diff
@@ -1,11 +1,3 @@
-declare module 'eslint' {
-  interface SharedConfigurationSettings {
-    unocss?: {
-      configPath?: string
-    }
-  }
-}
-
```

Comment:
This statement will affect the packaging failure of eslint-plugin and whether it can be removed.

<img width="552" alt="image" src="https://github.com/unocss/unocss/assets/57086651/f2d8e2a5-1272-473a-b549-6ed05e225146">